debian/patches/update-unit.patch

```diff
@@ -0,0 +1,23 @@
+Update ferm.service from archlinux
+
+https://projects.archlinux.org/svntogit/community.git/plain/trunk/ferm.service?h=packages/ferm
+
+Except, we still use sbin instead of bin.
+
+--- a/ferm.service
++++ b/ferm.service
+@@ -1,10 +1,14 @@
+ [Unit]
```

Comment:
As advised by https://wiki.debian.org/Teams/pkg-systemd/rcSMigration we should use `Wants=network-pre.target` and `Before=network-pre.target`. In addition `Before=shutdown.target